raiden/ui/console.py

```diff
@@ -216,27 +224,6 @@ def register_token(self, token_address_hex):
         self._raiden.register_channel_manager(channel_manager.address)
         return channel_manager
 
-    def ping(self, peer_address_hex, timeout=4):
```

Comment:
Added two api methods:

- `get_node_network_state` : returns the current know state for the given `node_address`
- `start_health_check_for` : starts a health check task for a node.